src/js/media-store/request-map.ts

```diff
@@ -161,10 +161,13 @@ export const requestMap: RequestMap = {
   [MediaUIEvents.MEDIA_UNMUTE_REQUEST](stateMediator, stateOwners) {
     const key = 'mediaMuted';
     const value = false;
-    // If we've unmuted but our volume is currently 0, automatically set it to some low volume
-    if (!stateMediator.mediaVolume.get(stateOwners)) {
-      stateMediator.mediaVolume.set(0.25, stateOwners);
-    }
+    const volumePref =
+      +globalThis.localStorage.getItem('media-chrome-pref-volume');
+    // If we've unmuted but the current volume is 0, restore the preferred volume or set it to some low volume
+    stateMediator.mediaVolume.set(
+      volumePref > 0.25 ? volumePref : 0.25,
+      stateOwners
+    );
```

Comment:
```suggestion
    // If we've unmuted but our volume is currently 0, automatically set it to some low volume
    if (!stateMediator.mediaVolume.get(stateOwners)) {
      stateMediator.mediaVolume.set(0.25, stateOwners);
    }
```

this actually introduces a regression because we're making volume 0.25 with every unmute request. even if volume was 1

Reply:
Actually this didn't set it to 0.25 if volume was 1. This only happened if `volumePref` was lower than 0.25. I made this UX decision so that the audio is clearly perceptible when unmuting and having a really low `volumePref`. But without this it works fine anyway